config/schemas/migrate.0.sql

```diff
@@ -6,7 +6,7 @@ CREATE TABLE device_info(device_id TEXT NOT NULL, is_registered INTEGER NOT NULL
 CREATE TABLE ecu_serials(serial TEXT UNIQUE, hardware_id TEXT NOT NULL, is_primary INTEGER NOT NULL CHECK (is_primary IN (0,1)));
 CREATE TABLE misconfigured_ecus(serial TEXT UNIQUE, hardware_id TEXT NOT NULL, state INTEGER NOT NULL CHECK (state IN (0,1)));
 CREATE TABLE primary_keys(private TEXT, public TEXT);
-CREATE TABLE installed_versions(hash TEXT UNIQUE, name TEXT NOT NULL);
+CREATE TABLE installed_versions(hash TEXT UNIQUE, name TEXT NOT NULL, is_current INTEGER NOT NULL CHECK (is_current IN (0,1)));
```

Comment:
You can't change existing migrations, because the devices in the field only apply migrations that they haven't seen before.  The answer here is to create a migrate.3.sql that adds the new column.